src/legacy/server/index_patterns/routes.ts

```diff
@@ -0,0 +1,104 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { first } from 'rxjs/operators';
+import { schema } from '@kbn/config-schema';
+import { InternalCoreSetup, KibanaRequest, RequestHandlerContext } from '../../../core/server';
```

Comment:
```suggestion
import { InternalCoreSetup, 
  APICaller, KibanaRequest, RequestHandlerContext } from '../../../core/server';
```

You may want to add these suggestions manually, they will throw typescript errors if committed as is here, I don't think suggestions work with multi-line though.

Reply:
I've fixed it with VS Code and prettier imported them in each line.